src/components/song/Song.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React from 'react'
+import PropTypes from 'prop-types'
+import './_Song.scss'
+import AlbumArt from '../common/album-art/AlbumArt'
 
-const Song = () => {
-  return <div>
-    <div className="imagen__container">
-      <img src="" alt="caratula" />
-    </div>
-    <div className="song__data">
-      <h3>Artists</h3>
-      <p>Name Song</p>
+const Song = ({ image, title, artist }) => {
+  return <div className="song">
+    <div className="song__content">
+      <AlbumArt width={35} height={35} image={image} />
```

Comment:
Add an alt attribute to the image 
What if instead of setting the image manually you pass the dimensions as props?
You can add default props if you want to set initial values